packages/kos-sdk/src/chains/tron/trc20.rs

```diff
@@ -0,0 +1,62 @@
+use web3::ethabi::Contract;
```

Comment:
can we use the same from ETH module as it is the same? maybe create a common ERC20 abi as this will repeat basically for every chain.